lib/rules/disallow-padding-newlines-in-blocks.js

```diff
@@ -1,17 +1,29 @@
 /**
  * Disallows blocks from beginning or ending with 2 newlines.
  *
- * Type: `Boolean`
+ * Type: `Boolean` or `Object`
  *
- * Value: `true` validates all non-empty blocks.
+ * Values:
+ *  - `true` validates all non-empty blocks.
+ *  - `Object`:
+ *    - `'open'` boolean value for disallowing extra padding newline after opening brace
+ *    - `'close'` boolean value for disallowing extra padding newline before closing brace
+ *    - `'allExcept'` array of exceptions:
+ *       - `'conditionals'` ignores conditional (if, else if, else) blocks
+ *       - `'functions'` ignores function blocks
  *
  * #### Example
  *
  * ```js
  * "disallowPaddingNewlinesInBlocks": true
+ * "disallowPaddingNewlinesInBlocks": { "open": true, "close": true }
+ * "disallowPaddingNewlinesInBlocks": { "open": true, "close": false }
+ * "disallowPaddingNewlinesInBlocks": { "open": false, "close": true }
+ * "disallowPaddingNewlinesInBlocks": { "allExcept": [ "conditionals" ] }
+ * "disallowPaddingNewlinesInBlocks": { "allExcept": [ "functions" ] }
```

Comment:
I would compress those examples, to 

``js
"disallowPaddingNewlinesInBlocks": true
"disallowPaddingNewlinesInBlocks": { "open": true, "close": true }
"disallowPaddingNewlinesInBlocks": { "allExcept": [ "conditionals" ] }
```

As suggested in https://github.com/jscs-dev/node-jscs/pull/1667/files#r37021702